lib/widgets/dialog.dart

```diff
@@ -296,16 +296,22 @@ class DialogTextField extends HookWidget {
       );
 }
 
-Future<bool> showConfirmMixinDialog(
+enum DialogEvent {
+  positive,
+  neutral,
+}
+
+Future<DialogEvent?> showConfirmMixinDialog(
```

Comment:
the semantics should be more explicit if add `negative` to `DialogEvent` and make this return value to no null.

Reply:
Cause `showMixinDialog` awalys return `Future<T?>`, we don't care whether click the `negative` button or quit the dialog.

Reply:
you are right, SGTM